migration/src/main/resources/META-INF/rewrite/axon-jakarta.yml

```diff
@@ -0,0 +1,74 @@
+  - org.openrewrite.java.ChangeType:
+      newFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.jpa.JpaDeadLetter
+      oldFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.legacyjpa.JpaDeadLetter
+  - org.openrewrite.java.ChangeType:
+      newFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.jpa.JpaSequencedDeadLetterQueue
+      oldFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.legacyjpa.JpaSequencedDeadLetterQueue
+  - org.openrewrite.java.ChangeType:
+      newFullyQualifiedTypeName: org.axonframework.eventhandling.tokenstore.jpa.JpaTokenStore
+      oldFullyQualifiedTypeName: org.axonframework.eventhandling.tokenstore.legacyjpa.JpaTokenStore
+  - org.openrewrite.java.ChangeType:
+      newFullyQualifiedTypeName: org.axonframework.eventsourcing.eventstore.EmbeddedEventStore
+      oldFullyQualifiedTypeName: org.axonframework.eventsourcing.eventstore.legacyjpa.EmbeddedEventStore
+  - org.openrewrite.java.ChangeType:
+      newFullyQualifiedTypeName: org.axonframework.eventsourcing.eventstore.jpa.JpaEventStorageEngine
+      oldFullyQualifiedTypeName: org.axonframework.eventsourcing.eventstore.legacyjpa.JpaEventStorageEngine
+  - org.openrewrite.java.ChangeType:
+      newFullyQualifiedTypeName: org.axonframework.eventsourcing.eventstore.jpa.SQLErrorCodesResolver
+      oldFullyQualifiedTypeName: org.axonframework.eventsourcing.eventstore.legacyjpa.SQLErrorCodesResolver
+  - org.openrewrite.java.ChangeType:
+      newFullyQualifiedTypeName: org.axonframework.messaging.interceptors.BeanValidationInterceptor
+      oldFullyQualifiedTypeName: org.axonframework.messaging.interceptors.legacyvalidation.BeanValidationInterceptor
+  - org.openrewrite.java.ChangeType:
+      newFullyQualifiedTypeName: org.axonframework.messaging.interceptors.JSR303ViolationException
+      oldFullyQualifiedTypeName: org.axonframework.messaging.interceptors.legacyvalidation.JSR303ViolationException
+  - org.openrewrite.java.ChangeType:
+      newFullyQualifiedTypeName: org.axonframework.modelling.command.GenericJpaRepository
+      oldFullyQualifiedTypeName: org.axonframework.modelling.command.legacyjpa.GenericJpaRepository
+  - org.openrewrite.java.ChangeType:
+      newFullyQualifiedTypeName: org.axonframework.modelling.saga.repository.jpa.JpaSagaStore
+      oldFullyQualifiedTypeName: org.axonframework.modelling.saga.repository.legacyjpa.JpaSagaStore
+  - org.openrewrite.java.ChangeType:
+      newFullyQualifiedTypeName: org.axonframework.springboot.autoconfig.JpaAutoConfiguration
+      oldFullyQualifiedTypeName: org.axonframework.springboot.autoconfig.legacyjpa.JpaAutoConfiguration
+  - org.openrewrite.java.ChangeType:
+      newFullyQualifiedTypeName: org.axonframework.springboot.autoconfig.JpaEventStoreAutoConfiguration
+      oldFullyQualifiedTypeName: org.axonframework.springboot.autoconfig.legacyjpa.JpaEventStoreAutoConfiguration
+  - org.openrewrite.java.ChangeType:
```

Comment:
I would expect the `org.axonframework.common.jpa.EntityManagerProvider` to `org.axonframework.common.legacyjpa.EntityManagerProvider` too here.

Reply:
I would expect the `org.axonframework.common.jpa.SimpleEntityManagerProvider` to `org.axonframework.common.legacyjpa.SimpleEntityManagerProvider` too here.

Reply:
Solved through the same ChangePackage as above:
```yaml
  - org.openrewrite.java.ChangePackage:
      oldPackageName: org.axonframework.common.legacyjpa
      newPackageName: org.axonframework.common.jpa
```